lib/common/alarm/CustomAlarmThreshold.ts

```diff
@@ -39,6 +39,18 @@ export interface CustomAlarmThreshold {
    */
   readonly alarmDescriptionOverride?: string;
 
+  /**
+   * Specifies how many samples (N) of the metric is needed to trigger the alarm.
+   * If this property is specified, an artificial composite alarm is created of the following:
```

Comment:
```suggestion
   * If this property is specified, an composite alarm is created of the following:
```

Reply:
If you remove "artificial", make sure to change the preceding "an" to an "a". :)